Kconfig

```
@@ -197,6 +197,11 @@ menu "LVGL configuration"
 				> 1 requires an operating system enabled in `LV_USE_OS`
 				> 1 means multiply threads will render the screen in parallel
 
+		config LV_DRAW_THREAD_STACKSIZE
+			int "stack size of draw thread"
```

LVGL has mix use of size unit, let's mention it's byte here.

```suggestion
			int "stack size of draw thread in byte"
```